packages/shoreline/src/components/filter/filter-value.tsx

```diff
@@ -13,14 +14,25 @@ export const FilterValue = forwardRef<HTMLSpanElement, FilterValueProps>(
     const filter = useFilterContext()
     const value = filter?.useState('value')
 
-    const isArray = Array.isArray(value)
-    const isLongArray = isArray && value.length > 1
+    const valueIsArray = Array.isArray(value)
+    const valueIsMultiSelected = valueIsArray && value.length > 1
+
+    const selectContext = useSelectContext()
+    const items = selectContext?.useState('items')
```

Comment:
Seems like this [was deprecated](https://ariakit.org/reference/use-select-store#usestate) in favor of [useStoreState](https://ariakit.org/reference/use-store-state)

Reply:
Basically the useStoreState does not exist yet in that version